requirements/base.txt

```diff
@@ -1,5 +1,5 @@
 django-environ~=0.10
-ds-caselaw-marklogic-api-client==29.0.1
+ds-caselaw-marklogic-api-client==29.1.0
```

Comment:
This should be able to go away with a rebase on `main`.

Reply:
Done.